resources/views/components/progress-bar/index.blade.php

```diff
@@ -8,12 +8,10 @@
     </span>
     @foreach ($checkoutSteps as $checkoutStepKey => $checkoutStep)
         <a href="{{ route('checkout', $checkoutStep) }}" @class([
-            'aspect-square w-3 rounded text-center',
-            'bg-ct-accent cursor-pointer' => $currentStepKey < $checkoutStepKey,
-            'bg-ct-border pointer-events-none' => $checkoutStepKey > $currentStepKey,
+            'size-3 rounded text-center bg-ct-accent',
+            'cursor-pointer' => $currentStepKey < $checkoutStepKey,
+            'pointer-events-none !bg-ct-border' => $checkoutStepKey > $currentStepKey,
             'outline-4 outline outline-ct-accent/20' => $checkoutStepKey === $currentStepKey
-        ])>
-            {{ $checkoutStepKey + 1 }}
```

Comment:
We don't need `{{ $checkoutStepKey + 1 }}` anymore?

Reply:
That element isn't designed. The steps + current steps are still there, but not within the progress dot. So just like the design originally